app/middlewares/CORSMiddleware.php

```diff
@@ -104,8 +104,13 @@ private function getAllowOriginHeaderValue(Request $request)
 
         // Check the domains using shell wildcard patterns
         $validCorsDomainFilter = function ($validCorsDomain) use ($origin) {
+            if (null === $origin) {
+                return false;
```

Comment:
This is against the `@return string|null` annotation. Can we return `null` instead so it fits the expected return value?
```suggestion
                return null;
```

Reply:
Yes indeed, that makes sense. I tested with it and it works as expected 👍